crates/holochain/src/core/workflow/countersigning_workflow/complete.rs

```diff
@@ -264,3 +292,69 @@ fn get_countersigning_op_hashes(
         )?
         .collect::<Result<Vec<_>, _>>()?)
 }
+
+/// When a countersigning session has become unresolvable automatically by querying peer state,
```

Comment:
We don't really have a concept of unresolvable exactly. There just "requires resolving" and "tried but could not resolve yet". I guess there are the cases of a split on the network where some AAAs think it was published and others don't. That is a possibility and we can't easily recover from that. However, in general it's just "not resolved yet" and we're allowing the user to make the decision that Holochain can't be certain of. I'd prefer to document it like that